x/subscription/spec/03_messages.md

```diff
@@ -10,11 +10,12 @@ Plan owners create plans, the message signer is the plan owner.
 
 ```protobuf
 message MsgCreateSubscriptionPlan {
-  string title;
-  string description;
+  string owner,  // signer
```

Comment:
string or some more specific type for address?

Reply:
in cosmos-sdk's proto files, it uses string for addresses.